src/equations/serre_green_naghdi_1d.jl

```diff
@@ -478,9 +553,17 @@ function rhs_sgn_upwind!(dq, q, equations, source_terms, cache, ::BathymetryFlat
                     p_x)
     end
 
-    # The code below is equivalent to
+    # add source term
+    # use dv as temporary storage to calculate source terms
+    fill!(dv, zero(eltype(dv)))
+    @trixi_timeit timer() "source terms" calc_sources!(dq, q, t, source_terms, equations,
+                                                       solver)
+    # add source terms to the right-hand side to be solved for
+    @.. tmp += dv
+
     #   dv .= (Diagonal(h) - D1mat_plus * Diagonal(1/3 .* h.^3) * D1mat_minus) \ tmp
```

Comment:
```suggestion
    # The code below is equivalent to
    #   dv .= (Diagonal(h) - D1mat_plus * Diagonal(1/3 .* h.^3) * D1mat_minus) \ tmp
```
Looks like this part of the comment was lost.